Documentation.app/Contents/MacOS/Library.playground/Pages/How to use nef library?.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,24 @@
+// nef:begin:header
+/*
+ layout: docs
+ title: How to use nef library?
```

Comment:
The title of the page should be changed accordingly, as well as possible links to this page.

```suggestion
 title: How to use nef as a library?
```